.github/workflows/go.yml

```
@@ -32,6 +32,9 @@ jobs:
       - name: Build
         run: CGO_ENABLED=0 find . -name go.mod -execdir go build ./... \;
 
+      - name: WASM Build
```

This is ambiguous with the new `wasip1` target, could we make it more explicit?
```suggestion
      - name: JS/WASM Build
```